### DISCUSSION TITLE
Nuxt 3 Content Security Policy

### DISCUSSION THREAD

**Discussion by baronkoko** [nuxt/nuxt#15963] [Questions] [answered]
Will the `CSP` (https://nuxtjs.org/docs/configuration-glossary/configuration-render/#csp)  option be added to the Nuxt 3 config? Or maybe there is some other option to implement it in Nuxt 3?

**Comment by pi0** [comment, marked as answer]
Hi! Linked issues:

- https://github.com/nuxt/framework/issues/591
- https://github.com/unjs/nitro/issues/81

**Reply by baronkoko** [reply]
Hi! Thank you very much!

**Reply by kliehm** [reply]
Any news on this? It seems the other ticket is idling.

**Reply by fkammer** [reply]
Is there currently any other way to be able to not set `'unsafe-inline'` for `script-src` and `style-src`? This is a huge security draw-back in my eyes.

**Reply by brendan1212** [reply]
> Is there currently any other way to be able to not set `'unsafe-inline'` for `script-src` and `style-src`? This is a huge security draw-back in my eyes.

Having the same issue here. Using Netlify for static.

**Reply by surgiie** [reply]
Wasnt this supported in nuxt 2? Why is it still not available in nuxt 3? :/
